### PR TITLE
fix: Auto log-out user instead of refreshing when token expires

### PR DESCRIPTION
**Pull request by AmandaMisjuwar**
## Notion ticket link

[Login Auth not working after token expires](https://www.notion.so/uwblueprintexecs/Login-Auth-refresh-not-working-after-token-expires-7e98633addd04560a6ffeb028d3a0fdc)

## Implementation description
* Auto log out user instead of doing the refresh (see the Community Fridge fix for the same issue: https://github.com/uwblueprint/community-fridge-kw/pull/193)

## Steps to test
1. Log in
2. After 1 hour, you should automatically be logged out

## What should reviewers focus on?
* 

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by AmandaMisjuwar** [comment]
Also, again I'm not able to run the linter so if somebody who's reviewing this can run yarn fix on the frontend that would be great :DD

**Review by dhmi19 — APPROVED** [review]
lgtm, i'll let @e-wai take a look before merging too

**Review by e-wai — APPROVED** [review]
Looks great, thanks so much for this 🙏

**Pull request merged**
